draft-irtf-cfrg-opaque.md

```diff
@@ -693,21 +695,25 @@ Input:
 Output:
 - envelope, the client's `Envelope` structure.
 - client_public_key, the client's AKE public key.
-- masking_key, an encryption key used by the server with the sole purpose 
+- masking_key, an encryption key used by the server with the sole purpose
   of defending against client enumeration attacks.
 - export_key, an additional client key.
 
-Steps:
-1. envelope_nonce = random(Nn)
-2. masking_key = Expand(randomized_pwd, "MaskingKey", Nh)
-3. auth_key = Expand(randomized_pwd, concat(envelope_nonce, "AuthKey"), Nh)
-4. export_key = Expand(randomized_pwd, concat(envelope_nonce, "ExportKey"), Nh)
-5. seed = Expand(randomized_pwd, concat(envelope_nonce, "PrivateKey"), Nseed)
-6. _, client_public_key = DeriveAuthKeyPair(seed)
-7. cleartext_creds = CreateCleartextCredentials(server_public_key, client_public_key, server_identity, client_identity)
-8. auth_tag = MAC(auth_key, concat(envelope_nonce, cleartext_creds))
-9. Create Envelope envelope with (envelope_nonce, auth_tag)
-10. Output (envelope, client_public_key, masking_key, export_key)
+def Store(randomized_pwd, server_public_key, server_identity, client_identity):
+  envelope_nonce = random(Nn)
+  masking_key = Expand(randomized_pwd, "MaskingKey", Nh)
+  auth_key = Expand(randomized_pwd, concat(envelope_nonce, "AuthKey"), Nh)
+  export_key = Expand(randomized_pwd, concat(envelope_nonce, "ExportKey"), Nh)
+  seed = Expand(randomized_pwd, concat(envelope_nonce, "PrivateKey"), Nseed)
+  _, client_public_key = DeriveAuthKeyPair(seed)
```

Comment:
While we are here, I kindof feel like it would be cleaner to do:
`(_, client_public_key) = DeriveAuthKeyPair(seed)`

instead of:
`_, client_public_key = DeriveAuthKeyPair(seed)`

for everywhere where DeriveAuthKeyPair is called.